dockyard/common/utils.py

```diff
@@ -0,0 +1,68 @@
+from oslo_config import cfg
+
+from dockyard.common import base, link
+from dockyard.common.container.scheduler.round_robin import RoundRobinScheduler
+
+from urllib3 import PoolManager
+
+rest_client = base.RESTClient()
+
+def get_config(group, option):
+    CONF = cfg.CONF
+    return CONF.group.option
+
+
+def get_host():
+    return RoundRobinScheduler().get_host()        
```

Comment:
It should be based on the driver from configuration file to be used for scheduling purpose.